src/test/java/co/rsk/bitcoinj/script/RedeemScriptParserFactoryTest.java

```diff
@@ -103,7 +110,7 @@ public void create_RedeemScriptParser_object_from_fast_bridge_p2sh_erp_multiSig_
         Assert.assertEquals(MultiSigType.FAST_BRIDGE_P2SH_ERP_FED, parser.getMultiSigType());
     }
 
-    @Test
+    @Test(expected = ScriptException.class)
     public void create_RedeemScriptParser_object_from_fast_bridge_P2SH_chunk() {
```

Comment:
Not sure the test name reflects what you are doing here. You are basically testing with a scriptSig, right?

Reply:
I'd go over these tests, do we really need to test with different script sigs?

Reply:
Done. Many tests were removed because they were redundant. Others were reorganized.